.github/workflows/spelling.yml

```diff
@@ -0,0 +1,36 @@
+name: Spell checking
+on:
+  pull_request_target:
+  push:
+  issue_comment:
+
+jobs:
+  build:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2
+    - uses: check-spelling/check-spelling@prerelease
+      with:
+        experimental_apply_changes_via_bot: 1
+        extra_dictionaries:
+          cspell:cpp/cpp.txt
+          cspell:golang/go.txt
+          cspell:node/node.txt
+          cspell:php/php.txt
+          cspell:aws/aws.txt
+          cspell:java/java.txt
+          cspell:npm/npm.txt
+          cspell:en_GB/src/wordsEnGb.txt
```

Comment:
Fair warning: this dictionary has a number of typos in it. But until this repository agrees to favor en-US, I'd rather tolerate some typos 🙂 . (I do plan to remove a bunch of them eventually, it's an in progress task.)